Update Readme with "Alternatives" (#33)
mhanberg/jekyll-postcss: README.md

@tailwind utilities;
```

> ## Alternatives 

Reviewer: ```suggestion
## Alternatives 

```

@tailwind utilities;
```

## Alternatives 

> To use Jekyll with Tailwindcss v2, the new [postcss-jekyll-v2](https://github.com/bglw/jekyll-postcss-v2) is probably a better choice.  

Reviewer: ```suggestion
- [postcss-jekyll-v2](https://github.com/bglw/jekyll-postcss-v2): works well with TailwindCSS 2.0  
```

> [bridgetown](https://github.com/bridgetownrb/bridgetown) - A Jekyll fork, which uses Weback for modern JS and CSS and is actively maintained. 

Reviewer: ```suggestion
- [bridgetown](https://github.com/bridgetownrb/bridgetown) - A Jekyll fork, which uses Weback for modern JS and CSS and is actively maintained. 
```

> - [bridgetown](https://github.com/bridgetownrb/bridgetown) - A Jekyll fork, which uses Weback for modern JS and CSS and is actively maintained. 

Reviewer: ```suggestion
- [bridgetown](https://github.com/bridgetownrb/bridgetown) - A Jekyll fork, which uses Webpack for modern JS and CSS and is actively maintained. 
```

> Instead of Jekyll, you may want to look into [bridgetown](https://github.com/bridgetownrb/bridgetown). It’s a Jekyll fork, which uses webpack for modern js and css. It is actively maintained.  

Reviewer: ```suggestion
[bridgetown](https://github.com/bridgetownrb/bridgetown) - A Jekyll fork, which uses Weback for modern JS and CSS and is actively maintained. 
```
